react-markdown-types-test.tsx

```diff
@@ -0,0 +1,7 @@
+import * as React from 'react'
+import * as ReactDom from 'react-dom'
+import * as ReactMarkdown from 'react-markdown'
+import * as ReactMarkdownWitHtml from 'react-markdown/with-html'
+
+ReactDom.render(<ReactMarkdown># header</ReactMarkdown>, document.getElementById('main'))
+ReactDom.render(<ReactMarkdownWitHtml># header</ReactMarkdownWitHtml>, document.getElementById('main'))
```

Comment:
will `document.body` or so also be fine for the tests? I like it because it’s a bit shorter and (while it’s all theoretically) would always work